torch/_higher_order_ops/scan.py

```diff
@@ -470,9 +426,9 @@ def scan_functionalize(ctx, combine_fn, init, xs, dim, reverse, additional_input
             functional_combine_fn,
             unwrapped_init,
             unwrapped_xs,
-            dim,
-            reverse,
-            unwrapped_additional_inputs,
+            dim=dim,
```

Comment:
Can we not sneak in small code-refactoring changes? It becomes kind of annoying to review many of these lol